eng/pipelines/coreclr/gcstress-extra.yml

```diff
@@ -33,6 +33,7 @@ jobs:
       testGroup: gcstress-extra
       liveLibrariesBuildConfig: Release
 
+# I assume we need to do something here?
```

Comment:
You don't have to do anything here as the testGroup is already flowed down as `gcstress-extra` to the template and then the change you will do under `run-test-job.yml` will be picked up.